manifests/a/Anki/Anki/2.1.63/Anki.Anki.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Anki.Anki
+PackageVersion: 2.1.63
+PackageLocale: en-US
+Publisher: Anki
+PublisherUrl: https://github.com/ankitects/anki
+PublisherSupportUrl: https://github.com/ankitects/anki/issues
+Author: Ankitects
+PackageName: Anki
+PackageUrl: https://apps.ankiweb.net
+License: AGPL-3.0-only

```

```suggestion
License: AGPL-3.0-or-later
```
According to LicenseUrl https://github.com/ankitects/anki/blob/main/LICENSE